pkg/ec2helper/ec2helper.go

```diff
@@ -1339,3 +1348,67 @@ func HasEbsVolume(image *ec2.Image) bool {
 
 	return false
 }
+
+func (h *EC2Helper) CreateUserLaunchTemplate(simpleConfig *config.SimpleInfo) (err error) {
+	launchIdentifier := uuid.New()
+
+	fmt.Println("Creating Launch Template...")
+	input := &ec2.CreateLaunchTemplateInput{
+		LaunchTemplateData: &ec2.RequestLaunchTemplateData{
+			ImageId:      &simpleConfig.ImageId,
+			InstanceType: &simpleConfig.InstanceType,
+			NetworkInterfaces: []*ec2.LaunchTemplateInstanceNetworkInterfaceSpecificationRequest{
+				{
+					AssociatePublicIpAddress: aws.Bool(true),
+					DeviceIndex:              aws.Int64(0),
+					Ipv6AddressCount:         aws.Int64(1),
+					SubnetId:                 aws.String(simpleConfig.SubnetId),
+				},
+			},
+		},
+		TagSpecifications: []*ec2.TagSpecification{
+			{
+				ResourceType: aws.String("launch-template"),
+				Tags: []*ec2.Tag{
+					{
+						Key:   aws.String("Name"),
+						Value: aws.String("Ec2-LaunchTemplate"),
+					},
+				},
+			},
+		},
+		LaunchTemplateName: aws.String(fmt.Sprintf("SimpleEC2LaunchTemplate-%s", launchIdentifier)),
+		VersionDescription: aws.String(fmt.Sprintf("Lauch Template %s", launchIdentifier)),
+	}
+
+	result, err := h.Svc.CreateLaunchTemplate(input)
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			fmt.Println(aerr.Error())
+		} else {
+			fmt.Println(err.Error())
+		}
+		return
+	}
+
+	simpleConfig.LaunchTemplateId = *result.LaunchTemplate.LaunchTemplateId
+	return
+}
+
+func (h *EC2Helper) DeleteUserLaunchTemplate(simpleConfig *config.SimpleInfo) (err error) {
```

Comment:
The only piece of data from `simpleConfig` that we need is the `LaunchTemplateId`, so it would be cleaner to pass just that data